src/Management/ManagementClient.php

```diff
@@ -526,4 +526,39 @@ public function findItemsFromTemporalIndex($hashId, $indexName, $params)
             throw ErrorHandler::create($e->getCode(), $e->getMessage(), $e);
         }
     }
+
+    /**
+     * Given a hashId, indexName and params, it gets an item list.
+     *
+     * @param string $hashId
+     * @param string $indexName
+     * @param array $params
+     * @return HttpResponseInterface
+     * @throws ApiException
+     */
+    public function findItems($hashId, $indexName, $params)
+    {
+        try {
+            return $this->itemsResource->findItems($hashId, $indexName, $params);
+        } catch (ApiException $e) {
+            throw ErrorHandler::create($e->getCode(), $e->getMessage(), $e);
+        }
+    }
```

Comment:
I think the public methods should accept directly the  fields that the request needs, in this case a param like `$ids` that shall be something like `["1", "2", ...]` and, another parameter could be present, something like `$extra_params` that can be merged with previously provided params

The way it is now, the person using the library, should guess what kind of params this request needs, the library doesn't throw any light of it.

This may worth for all other calls if there are like this, with a generic `$param` argument (but it can be fixed in other issue I guess)

```suggestion
    /**
     * Given a hashId, indexName and ids array of strings, it gets an item list.
     *
     * @param string $hashId
     * @param string $indexName
     * @param array $ids
     * @return HttpResponseInterface
     * @throws ApiException
     */
    public function findItems($hashId, $indexName, $ids)
    {
        try {
            return $this->itemsResource->findItems($hashId, $indexName, $ids);
        } catch (ApiException $e) {
            throw ErrorHandler::create($e->getCode(), $e->getMessage(), $e);
        }
    }
```

Reply:
I did it in that way to have the same structure as other methods.
In this case it could be a good way to put the parameters.
Maybe we need an accurate documentation, because the user don't know the input array structure :thinking:

Reply:
Documentation also is necessary yep, but as I said, can be in other PR